README.md

```diff
@@ -68,7 +68,7 @@ $ sudo usermod -a -G plugdev USERNAME
 ```
 
 #### Add udev rules
-Copy these two files into your /etc/udev.rules directory:
+Copy these two files into your /etc/udev/rules.d directory:
```

Comment:
If setup, there are only one director on Ubuntu 16.04 host; If update, you have both.